Define _XOPEN_SOURCE_EXTENDED to declare NCURSES_WIDECHAR for
widechar ncurses support with !glibc.

--- watch.c.orig	2015-03-17 12:04:11.025226853 +0100
+++ watch.c	2015-03-17 12:04:29.568180561 +0100
@@ -51,7 +51,9 @@
 #ifdef WITH_WATCH8BIT
 # include <wchar.h>
 # include <wctype.h>
+# define _XOPEN_SOURCE_EXTENDED
 # include <ncurses.h>
+# undef _XOPEN_SOURCE_EXTENDED
 #else
 # include <ncurses.h>
 #endif	/* WITH_WATCH8BIT */
